lib/furoshiki/jar_app.rb

```diff
@@ -158,7 +158,7 @@ def ensure_jar_exists
     # @param [Pathname, String] jar_path the location of the JAR to inject
     def inject_jar(jar_path)
       jar_dir = tmp_app_path.join('Contents/Java')
-      jar_dir.rmtree
+      jar_dir.rmtree if File.exist?(jar_dir)
```

Comment:
This line was breaking on my new template because that directory didn't previously exist...added a check to prevent exceptions.